### ISSUE TITLE
Request: Making giant centipede and summoned animals mountable

### ISSUE THREAD

**Issue by Nerevaristan** [Vek17/WrathMods-TabletopTweaks#74]
Would be a cool feature.

**Comment by TKDancer** [comment]
that would require existing animations, pretty sure if u try to force it so u can mount creatures we're not meant to mount u just stand inside them cause theres no animation for it

**Comment by Nerevaristan** [comment]
You are right but do you think it would be possible to make summoned animals mountable? The animation is already there.
Also Occultist is a Archetype that really fits the the theme.
